src/index.js

```diff
@@ -42,6 +41,10 @@ const patch = snabbdom.init([
 		votes.push(vote)
 	}
 
+	const commit = async () => {
+		await archive.commit()
```

Comment:
Thanks, didn't know that we need to do this with the [`DatArchive` API](https://beakerbrowser.com/docs/apis/dat.html), as it mimics [the `hyperdrive` API](https://www.npmjs.com/package/hyperdrive) which doesn't have this.

Reply:
Year previously, those changed only went into the staging and weren't commited